Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /testimonial-app
+WORKDIR 	/testimonial-app
+COPY requirements.txt /testimonial-app/
+RUN pip install -r requirements.txt
+RUN apt-get update && apt-get install vim -y --no-install-recommends
```

Comment:
highly not recommended to have an editor installed in docker. If you need to  debug, install it if you need for a one off case